Examples/Framework/include/ActsExamples/EventData/SimSpacePoint.hpp

```diff
@@ -28,15 +32,16 @@ class SimSpacePoint {
   /// @param varZ Measurement variance of the global longitudinal position
   /// @param measurementIndex Index of the underlying measurement
   template <typename position_t>
-  SimSpacePoint(const Eigen::MatrixBase<position_t>& pos, float varRho,
-                float varZ, Index measurementIndex)
+  SimSpacePoint(
```

Comment:
Why did we have that explicitly in `float`?

Reply:
Changed it to `ActsScalar`